Unable to run amplifylocal --version

I have just setup a localstack pro account and I am trying to deploy my existing amplify project to localstack. I have followed the setup instructions and run into this error when I try to run `amplifylocal --version`:

```
+ @aws-amplify/cli@4.41.0
added 1206 packages from 726 contributors and updated 6 packages in 49.448s
PS C:\Users\Emily Cheyne\source\repos\LocalSecurity\client-app> npm install -g amplify-js-local
C:\Users\Emily Cheyne\AppData\Roaming\npm\amplifylocal -> C:\Users\Emily Cheyne\AppData\Roaming\npm\node_modules\amplify-js-local\bin\amplifylocal
+ amplify-js-local@0.1.2
updated 1 package in 0.131s
PS C:\Users\Emily Cheyne\source\repos\LocalSecurity\client-app> amplifylocal --version
internal/modules/cjs/loader.js:960
  throw err;
  ^

Error: Cannot find module 'esm'
Require stack:
- C:\Users\Emily Cheyne\AppData\Roaming\npm\node_modules\amplify-js-local\bin\amplifylocal
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.<anonymous> (C:\Users\Emily Cheyne\AppData\Roaming\npm\node_modules\amplify-js-local\bin\amplifylocal:10:13)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\Emily Cheyne\\AppData\\Roaming\\npm\\node_modules\\amplify-js-local\\bin\\amplifylocal'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -11,7 +11,9 @@
     "organization": true
   },
   "license": "Apache-2.0",
-  "dependencies": {},
+  "dependencies": {
+    "esm": "^3.0.0"
+  },
   "repository": {
     "url": "https://github.com/localstack/amplify-js-local",
     "type": "git"

